docs/.vuepress/config.js

```diff
@@ -4,8 +4,17 @@ const examples = require('./containers/examples');
 const sourceCodeLink = require('./containers/sourceCodeLink');
 const nginxRedirectsPlugin = require('./plugins/generate-nginx-redirects');
 const assetsVersioningPlugin = require('./plugins/assets-versioning');
+const { getBuildDocsVersion, getLatestVersion } = require('./helpers');
 
 const buildMode = process.env.BUILD_MODE;
+const version = getBuildDocsVersion() || '**';
```

Comment:
With `version` I would be worried that the new developer might not understand why there are is `'**'`. They would have to search the source code to understand the meaning. I suggest changing the name to something like: 

```suggestion
const versionPath = getBuildDocsVersion() || '**';
```
Or maybe `versionPartialPath`?

Reply:
done in 219ea49